packages/grid/_modules_/grid/components/toolbar/GridFilterToolbarButton.tsx

```diff
@@ -11,9 +13,20 @@ import {
 import { gridPreferencePanelStateSelector } from '../../hooks/features/preferencesPanel/gridPreferencePanelSelector';
 import { GridPreferencePanelsValue } from '../../hooks/features/preferencesPanel/gridPreferencePanelsValue';
 import { optionsSelector } from '../../hooks/utils/optionsSelector';
+import { GridTranslationKeys } from '../../models/api/gridLocaleTextApi';
 import { GridApiContext } from '../GridApiContext';
 
+const useStyles = makeStyles((theme) => ({
+  root: {
+    '& ul': {
+      margin: theme.spacing(1),
+      padding: theme.spacing(0, 1),
+    },
+  },
+}));
```

Comment:
I think that we should name it, as it will be part of the public CSS API, see `GridColumnsPanel` for an example

Reply:
Updated